ledger/store/trackerdb/testsuite/accounts_kv_test.go

```diff
@@ -333,29 +380,29 @@ func CustomTestResourcesQueryAllLimited(t *customT) {
 	require.Equal(t, aidxResA0, prs[0].Aidx)
 	require.Equal(t, addrA, prs[0].Creator)
 	require.Equal(t, expectedRound, prs[0].Round) // db round (inside resources)
-	require.Equal(t, resDataA0, prs[0].Data)
+	require.Equal(t, resDataWithParamsA0AcctB, prs[0].Data)
 	require.Equal(t, expectedRound, rnd) // db round (from the return)
 
 	// Set min to 1, should return only 1 resource (index 1)
 	prs, rnd, err = aor.LookupLimitedResources(addrB, 1, 1, basics.AssetCreatable)
 	require.NoError(t, err)
 	require.Equal(t, 1, len(prs))
-	require.Equal(t, aidxResA1, prs[0].Aidx)
+	require.Equal(t, aidxResA2, prs[0].Aidx)
 	require.Equal(t, addrA, prs[0].Creator)
 	require.Equal(t, expectedRound, prs[0].Round) // db round (inside resources)
-	require.Equal(t, resDataA1, prs[0].Data)
+	require.Equal(t, resDataWithParamsA1AcctB, prs[0].Data)
 	require.Equal(t, expectedRound, rnd) // db round (from the return)
 
 	// Delete both resource creatables
```

Comment:
Prior to this, can you delete the app and verify this does not change the `LookupLimitedResources` response for both accounts? That would cover the account opted into a deleted app case (and verify the code filters out ctype = app)

Reply:
sure

Reply:
Check out the latest